frontend/yadava-dynasty/yadava.css

```
@@ -0,0 +1,1362 @@
+  align-items: center;
+  overflow: hidden;
+
+  background:
+    linear-gradient(
+      90deg,
+      rgba(5, 8, 12, 0.95),
+      rgba(5, 8, 12, 0.66),
+      rgba(5, 8, 12, 0.35)
+    ),
+    url("../assets/images/yadava-devagiri.jpg")
+    center / cover no-repeat;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)